src/js/utils/StructUtil.js

```diff
@@ -9,7 +10,7 @@ const StructUtil = {
    *                      One of Object, Array, Date, String, Number, or Boolean.
    * @return {Object}     Cloned Object without Structs
    */
-  copyRawObject: function copyRawObject(obj) {
+  copyRawObject: memoize(function copyRawObject(obj) {
```

Comment:
I have to say I'm not sold on this as it negates the very point of this function to begin with and seems highly risky.

I expect a copy function to always return a new copy such that `input !== output`.

We could be swapping one problem for another here. Accidental mutations are very real

Reply:
I agree and I'm not happy with this solution either. I'll see if there is a way to either completely remove the need for this function or adjust the plugin system to not waste that many cycles to copy objects that are never really used.